Make "Code-Review+1" blue when highlighted

fredriktomologic opened the pull request:
The action most likely to be the next to do is highlighted in Gerrit with a blue background.

Before: 
![before](https://cloud.githubusercontent.com/assets/3041463/19112498/634a381a-8b05-11e6-8dbf-27d3904ef83a.png)

After:
![after](https://cloud.githubusercontent.com/assets/3041463/19112497/63493f28-8b05-11e6-95a2-6ced9be4f655.png)

StephenKing committed:
Fix URL to JS file

Commit:
Make "Code-Review+1" blue when highlighted

The action most likely to be the next to do is highlighted in Gerrit with a blue background.

fredriktomologic committed:
Merge pull request #1 from StephenKing/StephenKing-patch-1

Fix URL to JS file

shellscape commented on dist/octogerrit.css:
```diff
@@ -668,8 +668,11 @@ div.com-google-gerrit-client-change-ChangeScreen_BinderImpl_GenCss_style-idAndSt
   width: 100%;
 }
+button.com-google-gerrit-client-change-ChangeScreen_BinderImpl_GenCss_style-highlight {
+  background-image: -webkit-linear-gradient(top, #4d90fe, #4d90fe);
```
check out github's 'Start a Review' and 'Review Changes' buttons. they've got a nice, subtle gradient shift from top to bottom with a border color change to match. I'd like to see this change follow suit so we can keep the elements of the ux in the same spirit and feel.

shellscape reviewed (changes requested):
I'd like to see a gentler button color there. If that's not something you'd like to play with, I can merge and update it.

fredriktomologic commented:
Good input. I think you have a better understanding of what you want. I'd be happy if you would merge and fix. Thank you!

Commit:
Update README to inform about the Tomologic fork

Tomologic has some specific changes that might not suite the upstream initiative.
We'll keep them in this repo.

Commit:
Unhide some fields we need

We like all functionality to work, even if it's ugly.

Commit:
Fix URL to JS file

This time in the source file as well.
